ec2/InstanceType.dhall

```diff
@@ -389,70 +410,268 @@ let alternatives =
+
+                          in    [ InstanceType.m5a-xlarge
+                                , InstanceType.r5a-xlarge
+                                ]
+                              # ( if        _params.permitLowerClassCPU
+                                        &&  _params.permitWorseNetwork
+                                  then  lowClass
+                                  else  [] : List InstanceType
+                                )
+
+                    let instancefamily =
+                          [ InstanceType.c5-xlarge
+                          , InstanceType.c5d-xlarge
+                          , InstanceType.c5n-xlarge
+                          ]
+
+                    in    instancefamily
+                        # ( if    _params.permitCrossInstanceFamily
+                            then  cross-instances
+                            else  [] : List InstanceType
+                          )
+                , c5a-2xlarge =
+                    let cross-instances =
+                          let lowClass =
+
+                              {-
+                                lowClass - all instance types that have lower ECU and lower network performance
+                                  -}
+                                [ InstanceType.m4-2xlarge ]
+
+                          in    [ InstanceType.m5a-2xlarge
+                                , InstanceType.r5a-2xlarge
+                                ]
+                              # ( if        _params.permitLowerClassCPU
+                                        &&  _params.permitWorseNetwork
+                                  then  lowClass
+                                  else  [] : List InstanceType
+                                )
+
+                    let instancefamily =
+                          [ InstanceType.c5-2xlarge
+                          , InstanceType.c5d-2xlarge
+                          , InstanceType.c5n-2xlarge
+                          ]
+
+                    in    instancefamily
+                        # ( if    _params.permitCrossInstanceFamily
+                            then  cross-instances
+                            else  [] : List InstanceType
+                          )
+                , c5a-4xlarge =
+                    let cross-instances =
+                          let lowClass =
+
+                              {-
+                                lowClass - all instance types that have lower ECU and lower network performance
+                                  -}
+                                [ InstanceType.m4-4xlarge ]
+
+                          in    [ InstanceType.m5a-4xlarge
+                                , InstanceType.r5a-4xlarge
+                                ]
+                              # ( if        _params.permitLowerClassCPU
+                                        &&  _params.permitWorseNetwork
+                                  then  lowClass
```

Comment:
You need to have permitLowerEBSBandwidth here too because c5a.4xlarge has EBS bandwidth up to 3170 Mbps while m4.4xlarge has EBS bandwidth of 2000 Mbps dedicated